lib/config/options.rb

```diff
@@ -122,7 +122,7 @@ def merge!(hash)
     def [](param)
       return super if SETTINGS_RESERVED_NAMES.include?(param)
       return super if RAILS_RESERVED_NAMES.include?(param)
-      send("#{param}")
+      public_send("#{param}")
     end
 
     def []=(param, value)
```

Comment:
Any thoughts on whether or not it makes sense to use `public_send` in `[]=` as well?

Reply:
I think it doesnt make sense. Private setter doesnt make sense
And I dont know any private method like "\/.+=\" in the chain of inheritance
`Config::Options.ancestors.any? { |c| c == ::Class ? c.private_instance_methods.any? {|m| m[/.+=/]} : false }`